release/lib/update-dependencies.ts

```diff
@@ -15,6 +19,77 @@ const propertiesToUpdate = [
     'peerDependencies'
 ];
 
+const nonOptionalDependencyTypes = [
```

Comment:
Nit: They are properties, what about change the name to `nonOptionalDependencyProperties`? (or any reduce version of that name hehehe)

Reply:
Sure.